### ISSUE TITLE
Larger Buttons

### ISSUE THREAD

**Issue by jporterscienceworld** [ScienceWorldCA/domelights#21]
The buttons on the right could be bigger, three across instead of 4. That way it will take up more space vertically and there won’t be a massive blank area at the bottom

**Comment by scienceworld** [comment]
![image](https://cloud.githubusercontent.com/assets/3997916/6495673/153b6e80-c281-11e4-8786-33a987a42afe.png)

**Comment by RobertButterworth** [comment]
Yep I agree, It could limit the number of brushes that we can display though, but we could get around that with grouping. What is the lowest resolution that we will be supporting?

Fire is the most complex brush in regards to UI atm, how do you see its options being placed?

**Comment by scienceworld** [comment]
colour controls could simply be controls.  the items in the panel could change depending on the selected pattern or effect, but then always be in a consistent place.
